rewrite-xml/src/main/java/org/openrewrite/xml/trait/SpringTypeOrPackageReference.java

```diff
@@ -30,16 +28,22 @@
 import java.util.regex.Pattern;
 
 @Value
-class SpringTypeReference implements TypeReference {
+class SpringTypeOrPackageReference implements Reference {
```

Comment:
With `Kind` being a property of `Reference` I wonder if we should have `TypeOrPackage` in the name here, or opt for `SpringReference` instead.

Reply:
Not related to Tim's comment: Looking at https://www.springframework.org/schema/beans/spring-beans-4.3.xsd, I think we should also match the `key-type` and `value-type` attributes in some context in any case.